### PR TITLE
fix(modal): removed drop shadow, updated colors

### PR DESCRIPTION
**Pull request by mattnolting**
fixes https://docs.google.com/spreadsheets/d/1fBCPPb4Z1sqUDKgQkpueGe8VQZo4pBUA9PV4vnFCnDw/edit#gid=0&range=A5

### CODE REVIEW

**Review by spadgett — COMMENTED** [review]
/assign @rhamilto @sg00dwin 
/ok-to-test

File: frontend/public/components/modals/_modals.scss [hunk 1]
```diff
@@ -35,29 +40,23 @@
   min-height: 100%;
   // --pf-c-modal-box--Padding* will not render correctly here so applying pf4 spacer values
   padding: 0 var(--pf-global--spacer--xl) var(--pf-global--spacer--xl);
-  @include scroll-shadows-vertical-covers;
```
Comment (sg00dwin) [inline frontend/public/components/modals/_modals.scss:38]: Delete corresponding `@mixin` file 
frontend/public/style/mixin/_scroll-shadows.scss

Comment (sg00dwin) [inline frontend/public/components/modals/_modals.scss:38]: Delete corresponding `@mixin` file 
frontend/public/style/mixin/_scroll-shadows.scss

File: frontend/public/components/modals/_modals.scss [hunk 2]
```diff
@@ -19,14 +29,9 @@
   overflow-y: auto;
   padding: 0;
   position: relative;
-  @include scroll-shadows-vertical;
   -webkit-overflow-scrolling: touch;
 }
 
-.modal-body-border {
-  border-top: 1px solid $pf-color-black-300;
```
Comment (sg00dwin) [inline frontend/public/components/modals/_modals.scss:27]: The original operator install modal designs included a line between the heading and scrolling body.
https://github.com/itsptk/openshift-origin-design/blob/3cbbabfc06819bea6eadb05dc85e49d218ad2eab/designs/administrator/olm/select-install-operator-version/index.md

**Comment by sg00dwin** [comment]
A couple of comments. Overall looks good.

File: frontend/public/components/modals/_modals.scss (hunk 2 above)
Comment (mattnolting) [reply frontend/public/components/modals/_modals.scss]: @sg00dwin updated

File: frontend/public/components/modals/_modals.scss [hunk 3]
```diff
@@ -35,29 +44,23 @@
...
       max-width: 100%;
 
       &[disabled],
       &[readonly] {
-        --pf-c-form-control--BackgroundColor: rgba(128, 128, 128, 0.15);
+        --pf-c-form-control--BackgroundColor: var(--pf-global--disabled-color--300);
       }
     }
   }
 }
```
Comment (sg00dwin) [inline frontend/public/components/modals/_modals.scss:60]: Lines 49 thru 60 should be deleted, they only supported scroll-shadows.

**Comment by yanpzhan** [comment]
@mattnolting could you help list some related page to check for the update? Thanks

**Comment by mattnolting** [comment]
@yanpzhan the `.modal-body-inner-shadow-covers` element contained `padding`, [removing it](https://github.com/openshift/console/pull/11148#discussion_r823986483) also removes the padding. To correct I've added the padding to `.modal-body-content`

Screen Shot 2022-03-15 at 9 47 57 AM

Screen Shot 2022-03-15 at 9 41 04 AM

Screen Shot 2022-03-15 at 9 40 48 AM

Screen Shot 2022-03-15 at 9 39 11 AM

**Review by spadgett — APPROVED** [review]
/approve
/label docs-approved
/label px-approved

We have docs- and px-approval for the epic.

**Comment by yanpzhan** [comment]
Tested against the pr, the modal box has border issue, texts in all modals are too near to the left and right side of modal.
@mattnolting could you have a check?
Screenshot from 2022-03-22 10-13-21
Screenshot from 2022-03-22 10-13-47
Screenshot from 2022-03-22 10-11-46

**Comment by mattnolting** [comment]
@yanpzhan Thanks for looking at this! Here's what I'm seeing:

Screen Shot 2022-03-22 at 2 50 05 PM

Screen Shot 2022-03-22 at 2 48 41 PM

Screen Shot 2022-03-22 at 2 48 31 PM

Screen Shot 2022-03-22 at 2 41 51 PM

I rebased, updated and force pushed. Hopefully that will resolve the issues you're seeing.

**Comment by yanpzhan** [comment]
Retested the pr after the codes are rebased, now all modals work well in dark theme.
/label qe-approved

**Comment by rhamilto** [comment]
@mattnolting, please squash your commits.

@sg00dwin, PTAnotherL

**Comment by rhamilto** [comment]
Unrelated to the changes here, but we also have spacing issues in modals (example below).  Are these issues being tracked/fixed somewhere?

Screen Shot 2022-03-30 at 11 36 16 AM

**Comment by mattnolting** [comment]
> Unrelated to the changes here, but we also have spacing issues in modals (example below). Are these issues being tracked/fixed somewhere?

@rhamilto I don't believe so. I'd be happy to address in a followup!

**Comment by rhamilto** [comment]
> Unrelated to the changes here, but we also have spacing issues in modals (example below). Are these issues being tracked/fixed somewhere?
> 
> Screen Shot 2022-03-30 at 11 36 16 AM

Another instance I noticed:

Screen Shot 2022-04-12 at 8 49 42 AM

**Pull request merged**